### PR TITLE
Issue #210840 Task Make Storylingo App Responsive Should work on Mobile and Tablet

### PR DESCRIPTION
**Pull request by ajinkyapandetekdi**

### CODE REVIEW

File: src/components/Game.js [hunk 1]
```diff
@@ -196,10 +196,10 @@ function Game() {
     <div className="main-container">
       <div className="top-header">
         <img
+        className="storylingo-img"
           src={logo}
-          height="25px"
+          // height="25px"
```
Comment (gouravmore) [inline src/components/Game.js:201]: remove the comment if not required

File: src/components/Game.js [hunk 2]
```diff
@@ -196,10 +196,10 @@ function Game() {
     <div className="main-container">
       <div className="top-header">
         <img
+        className="storylingo-img"
```
Comment (gouravmore) [inline src/components/Game.js:199]: the class name should be specific

File: src/components/Game.js [hunk 3]
```diff
@@ -196,10 +196,10 @@ function Game() {
...
           src={logo}
-          height="25px"
+          // height="25px"
           alt="logo"
-          style={{ cursor: "pointer" }}
```
Comment (gouravmore) [inline src/components/Game.js:202]: pointer not required?

File: src/components/Game.js [hunk 4]
```diff
@@ -412,6 +412,7 @@ function Game() {
             <div style={{ position: "relative" }}>
               <img
                 // height="13px"
+                className="img-responsive"
```
Comment (gouravmore) [inline src/components/Game.js:415]: will this class name can be applied for all images?

File: src/index.css [hunk 5]
```diff
@@ -183,8 +186,31 @@ h1 {
...
+  .position-set{
+    margin-top: -18px;
+  }
+
+  .story-text {
+    text-align: center;
+    font-size: 18px;
+    font-weight: bold;
+    width: 90%;
+    padding-left: 24px !important;
```
Comment (gouravmore) [inline src/index.css:199]: !important is required here?

File: src/index.css [hunk 6]
```diff
@@ -183,8 +186,31 @@ h1 {
...
+    padding-right: 45px;
+   }
+   .storylingo-img{
+    cursor: pointer;
+    height: 15px;
```
Comment (gouravmore) [inline src/index.css:204]: should not be in pixcel

File: src/index.css [hunk 7]
```diff
@@ -183,8 +186,31 @@ h1 {
...
+  }
+
+  .player_text{
+    font-size: 20px;
+    margin-top: 50px;
```
Comment (gouravmore) [inline src/index.css:209]: margin required?

**Review by gouravmore — CHANGES_REQUESTED** [review]

File: src/index.css (hunk 7 above)
Comment (ajinkyapandetekdi) [reply src/index.css:209]: Yes, in responsive screen Player Text is Overlapping with Storylingo Logo

File: src/index.css (hunk 6 above)
Comment (ajinkyapandetekdi) [reply src/index.css:204]: It's was already declared in Pixel. I just moved inline CSS to external css.

File: src/index.css (hunk 5 above)
Comment (ajinkyapandetekdi) [reply src/index.css:199]: Not necessary. Removed It

File: src/components/Game.js (hunk 1 above)
Comment (ajinkyapandetekdi) [reply src/components/Game.js]: removed

File: src/components/Game.js (hunk 3 above)
Comment (ajinkyapandetekdi) [reply src/components/Game.js:202]: Added In External CSS

File: src/components/Game.js (hunk 4 above)
Comment (ajinkyapandetekdi) [reply src/components/Game.js:413]: not used Anywhere removed it

File: src/components/Game.js (hunk 2 above)
Comment (ajinkyapandetekdi) [reply src/components/Game.js:199]: I changed the name. Now it's storyling-logo. 
or You can suggest me.

**Pull request merged**